.github/workflows/ci.yml

```diff
@@ -298,10 +299,27 @@ jobs:
           # Include a test for the P2P mode
           - mode: "p2p"
             era: ${{ fromJSON(needs.build-ubuntu-X64.outputs.eras)[0] }}
+            next_era: [""]
             cardano_node_version: "9.1.1"
             hard_fork_latest_era_at_epoch: 0
             run_id: "#1"
             extra_args: "--use-p2p-network"
+          # Include a test for the era switch without regenesis
+          - mode: "p2p"
+            era: ${{ fromJSON(needs.build-ubuntu-X64.outputs.eras)[0] }}
+            next_era: ${{ fromJSON(needs.build-ubuntu-X64.outputs.eras)[1] }}
```

Comment:
Shouldn't the era switch tests without regenesis and with regenesis be executed for the default `std` mode as well?

Reply:
Good point, we only need to test them in `std` :+1: